### ISSUE TITLE
Rails 6.1 undefined method `[]' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by ahmgeek** [thoughtbot/clearance#923]
When I try to run `rails g migration clearance:install`
I get: 
```
	 1: from /Users/ahmgeek/.rvm/gems/ruby-2.7.2/gems/clearance-2.3.0/lib/generators/clearance/install/install_generator.rb:126:in `migration_primary_key_type_string'
/Users/ahmgeek/.rvm/gems/ruby-2.7.2/gems/clearance-2.3.0/lib/generators/clearance/install/install_generator.rb:132:in `configured_key_type': undefined method `[]' for nil:NilClass (NoMethodError)```

**Comment by max2320** [comment]
I had the same issue and placed this PR https://github.com/thoughtbot/clearance/pull/924 with a fix

**Comment by eebs** [comment]
Thanks for reporting this issue, closed in eeb5d4007833af7e5332f2e0fa6e1cf2cb653af5.
